### DISCUSSION TITLE
group level requirement

### DISCUSSION THREAD

**Discussion by Moscano** [AscensionGameDev/Intersect-Engine#1477] [Ideas]
**Is your feature request related to a problem? Please describe.**

One suggestion, group level requirement, for a level 100 not to be able to share the experience with another level 10 player.

**Describe the solution you'd like**

Example: 10 level difference level requirement, or 10% difference experience requirement.

**Additional context**

**Comment by WeylonSantana** [comment]
A while ago there were options to limit the distance between party invitations, but it was removed, because it limits the creation of a party, I think the same thing about this one. A lot of people would want this (I could use it), but overall, we'd be forcing players to obey us, in a (sometimes) nonsensical order, so I'm not much for it.

I've seen many people level up weak players, with huge level differences, sometimes it's just what strong players have to do in the game after going through all the content, so i think "not add".
